### ISSUE TITLE
(Bug report) No longer to set start of index or choose style for list

### ISSUE THREAD

**Issue by Piztou** [zadam/trilium#4563]
### Trilium Version

0.62.4

### What operating system are you using?

Windows

### What is your setup?

Local + server sync

### Operating System Version

Windows 10 version 22H2

### Description

Hello,
There used to be a way to set the start of index for numbered lists (introduced in v 0.57 , #3308 ), as well as set the style of markers for ordered and unordered lists, as demonstrated there : https://ckeditor.com/docs/ckeditor5/latest/features/lists/lists.html#list-styles
 
but those options are no longer available.

I used those 2 tools pretty often, and was wondering if it would be possible to enable them again.

**Comment by zadam** [comment]
Hello, this plugin has been deactivated in recent versions, because it crashes the editing component (for unclear reasons).

**Comment by CobriMediaJulien** [comment]
Hey Zadam,

is there a way to have a setting in the option menu for "experimental functions" and to let the user activate CKE PlugIns at own risk?

**Comment by zadam** [comment]
No, this is not possible. Currently it's not possible to get this plugin back.

**Comment by eliandoran** [comment]
The plugin was re-enabled so it should be OK now.
